pylint_odoo/checkers/no_modules.py

```diff
@@ -213,8 +221,25 @@ class NoModuleChecker(BaseChecker):
         }),
     )
 
+    def visit_binop(self, node):
+        # First, check if the operator '%' exists,
+        # then obtain the left and right nodes.
+        if node.op == '%':
+            left_node = node.left
+            args = node.right
+            # In order to check if the left node is a constant and string,
+            # if it is True, then it is assumed that the left node is a query.
+            if (isinstance(left_node, astroid.Const) and
```

Comment:
We don't need a constant to check that is a `sql_injection`
If exists a `%` in a `*cr.execute()` method is a `sql_injection` too.

e.g.
```python
var = "select * from table where id in %s"
values = ([1,2,3,],)
cr.execute(var%values)  # sql injection too
```

Could you add this case to your tests, please?